tests/test_stepfunctions/test_stepfunctions.py

```diff
@@ -797,12 +805,32 @@ def test_state_machine_stop_execution():
     stop["stopDate"].should.be.a(datetime)
 
 
+@mock_stepfunctions
+@mock_sts
+def test_state_machine_stop_raises_error_when_unknown_execution():
+    client = boto3.client("stepfunctions", region_name=region)
+    client.create_state_machine(
+        name="test-state-machine",
+        definition=str(simple_definition),
+        roleArn=_get_default_role(),
+    )
+    with pytest.raises(ClientError) as ex:
+        unknown_execution = (
+            "arn:aws:states:"
+            + region
+            + ":"
+            + _get_account_id()
+            + ":execution:test-state-machine:unknown"
+        )
+        client.stop_execution(executionArn=unknown_execution)
+    ex.value.response["Error"]["Code"].should.equal("ExecutionDoesNotExist")
+    ex.value.response["Error"]["Message"].should.contain("Execution Does Not Exist:")
+
+
 @mock_stepfunctions
 @mock_sts
 def test_state_machine_describe_execution_after_stoppage():
-    account_id
```

Comment:
Ehm.. what. Why would anyone have this in a test.
Nice find :)

Reply:
Hah, just raised an eyebrow when I saw it!